src/lib/components/schedule/schedule-form-view.svelte

```diff
@@ -108,11 +116,9 @@
     </header>
     <form class="mb-4 flex w-full flex-col gap-4 md:w-2/3 xl:w-1/2">
       {#if $error}
-        <p
-          class="rounded-md border-2 border-orange-500 bg-orange-100 p-5 text-center"
-        >
+        <Alert intent="error" title="" bold>
```

Comment:
Not a huge fan of this, but I think it's ok for now if you wouldn't mind making a ticket to make `title` on `<Alert />` an optional prop?

Reply:
Done ✅ https://github.com/temporalio/ui/pull/1377